config.py

```diff
@@ -460,8 +460,8 @@ def create_transaction(db):
   # The whitelist of client IDs for OAuth applications that allow for direct login.
   DIRECT_OAUTH_CLIENTID_WHITELIST = []
 
-  # URL that specifies the location of the prometheus stats aggregator.
-  PROMETHEUS_AGGREGATOR_URL = 'http://localhost:9092'
+  # URL that specifies the location of the prometheus pushgateway.
+  PROMETHEUS_PUSHGATEWAY_URL = 'http://localhost:9091'
```

Comment:
What was the reason for changing from 9092 to 9091? (I'd like to document in release notes.)

Reply:
9091 is the official reserved port for pushgateway. They explicitly don't use 9092 for anything because it conflicts with Kafka and one other protocol. [Check out the wiki](https://github.com/prometheus/prometheus/wiki/Default-port-allocations#core-components-starting-at-9090).

Reply:
9091 is the officially reserved port for pushgateway. 9092 is actually completely passed over by the Prometheus developers because it conflicts with Kafka.

https://github.com/prometheus/prometheus/wiki/Default-port-allocations#core-components-starting-at-9090

Reply:
Thanks